docs/reference/components.rst

```diff
@@ -1,10 +1,43 @@
 Components
-**********
+==========
 
 .. include:: /reuse/components-intro.rst
 
-Components employ a `Craft Parts`_ feature called ``partitions``. See
-:ref:`component and partitions<components-and-partitions>` for an explanation
-on how they are related.
+Like snaps, components are packaged a squashfs file. When installed, the
+component is mounted under ``$SNAP/../components/$SNAP_REVISION/`` so
```

Comment:
We could recommend `/snap/$SNAP_INSTANCE_NAME/components/$SNAP_REVISION`.  Neither are user-friendly.  My note here glosses over that this is a symlink to another directory where the component is actually mounted.

I asked in the google doc if there's a recommended way for a snap to access this.

Reply:
It's an actual glob to get to the component files from $SNAP.

`/snap/$SNAP_INSTANCE_NAME/components/$SNAP_REVISION` works too, but neither option is user-friendly.  My note here glosses over that this is a symlink to another directory where the component is actually mounted.

I asked in the google doc if there's a recommended way for a snap to access component files.

Reply:
OK, with that in mind, I agree that's a mouthful to explain. Do you think this would be slightly clearer?

```suggestion
component is mounted at the path ``$SNAP/../components/$SNAP_REVISION/`` so
```